CONTRIBUTING.md

```diff
@@ -72,20 +77,23 @@ mvn org.eclipse.dash:license-tool-plugin:license-check -Ddash.iplab.token=$ECLIP
 
 For more information on the tool and how to acquire the token, check https://github.com/eclipse/dash-licenses.
 
-## General contribution to the project
+## General Contribution to the Project
+
+General contributions e.g. contributions to improve documentation are welcome.
```

Comment:
I would not focus on documentation - general contribution is not limited to this. I would not recommend explaining this too deeply. General link to issues and discussions are sufficient. In case labels or other things change these links were useless.